sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -729,12 +902,15 @@ public void setValueDeserializer(String valueDeserializer) {
 
     /**
      * Provide custom {@link TimestampPolicyFactory} to set event times and watermark for each
-     * partition. {@link TimestampPolicyFactory#createTimestampPolicy(TopicPartition, Optional)} is
-     * invoked for each partition when the reader starts.
+     * partition when beam_fn_api is disabled. {@link
```

Comment:
I think we can keep the original comment here since we support the timestamp policy fn now.